activesupport/test/message_verifier_test.rb

```diff
@@ -202,7 +224,14 @@ def verifier_options
 class MessageVerifierMetadataJSONTest < MessageVerifierMetadataTest
   private
     def verifier_options
-      { serializer: MessageVerifierTest::JSONSerializer.new }
+      { serializer: ActiveSupport::JSON}
```

Comment:
Should this be our new JSON class ?

Reply:
nvm, I see what is happening 😄

Reply:
It should be `ActiveSupport::JsonWithMarshalFallback`